fuzzing/config/config.go

```diff
@@ -140,11 +143,38 @@ type OptimizationTestingConfig struct {
 	TestPrefixes []string `json:"testPrefixes"`
 }
 
+// LoggingConfig describes the configuration options for logging to console and file
+type LoggingConfig struct {
+	// Level describes whether logs of certain severity levels (eg info, warning, etc.) will be emitted or discarded.
+	// Increasing level values represent more severe logs
+	Level zerolog.Level `json:"level"`
```

Comment:
I think we should just quickly add validation for this field before merging it, for a nicer error. Currently if you run it you'll get a vague error that doesn't tell you it's regarding logging:
```
error Failed to run the fuzz command error="Unknown Level String: 'blah', defaulting to NoLevel"
```

Reply:
Unfortunately, this occurs during an unmarshaling operation and cannot be caught early. resolving...